import_export_extensions/resources.py

```diff
@@ -40,6 +41,47 @@ class TaskState(Enum):
     PARSING = _("Parsing")
 
 
+class SkippedRow(results.RowResult):
+    """Custom row result class with ability to store skipped errors in row."""
+    def __init__(self):
+        self.non_field_skipped_errors: list[Error] = []
+        self.field_skipped_errors: dict[str, list[ValidationError]] = dict()
+        super().__init__()
+
+    @property
+    def has_skipped_errors(self) -> bool:
+        """Return True if row contain any skipped errors."""
+        if len(self.non_field_skipped_errors) > 0 or len(self.field_skipped_errors) > 0:
+            return True
+        return False
+
+    @property
+    def skipped_errors_count(self) -> int:
+        """Return count of skipped errors."""
+        return (
+            len(self.non_field_skipped_errors)
+            + len(self.field_skipped_errors)
+        )
+
+
+class ResultWithSkippedRows(results.Result):
+    """Custom result class with ability to store info about skipped rows."""
+
+    @property
+    def has_skipped_rows(self) -> bool:
+        """Return True if contain any skipped rows."""
+        if any(row.has_skipped_errors for row in self.rows):
+            return True
+        return False
+
+    @property
+    def skipped_rows(self) -> list[SkippedRow]:
+        """Return all rows with skipped errors."""
+        return list(
+            filter(lambda row: row.has_skipped_errors, self.rows),
+        )
+
+
```

Comment:
1. I think that classes related to result must be in separate file
2. What do you think to name them as original, i.e. `RowResult` and `Result`? `ResultWithSkippedRows` object may actually not contain skipped rows